test/e2e/framework/framework.go

```diff
@@ -509,6 +558,19 @@ func increaseAzureInstanceSize(rawProviderSpec *runtime.RawExtension, providerCo
 	return nil
 }
 
+// tagOpenStackProviderSpec makes a low-impact change to the OpenStack providerSpec that is supposed to trigger the redeployment of the Control Plane.
+func tagOpenStackProviderSpec(rawProviderSpec *runtime.RawExtension, providerConfig providerconfig.ProviderConfig) error {
+	cfg := providerConfig.OpenStack().Config()
+
+	cfg.Tags = append(cfg.Tags, "ccpmso-test-tag")
```

Comment:
Nit: other places we've similar things, we've just used `cpms` rather than `ccpmso`
```suggestion
	cfg.Tags = append(cfg.Tags, "cpms-test-tag")
```